tests/ssl-opt.sh

```diff
@@ -8655,11 +8655,11 @@ run_test    "TLS1.3: Not supported version check: tls1_2 and tls1_3" \
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
 run_test    "TLS1.3: handshake dispatch test: tls1_3 only" \
-            "$P_SRV min_version=tls1_3 max_version=tls1_3" \
-            "$P_CLI min_version=tls1_3 max_version=tls1_3" \
+            "$P_SRV debug_level=2 min_version=tls1_3 max_version=tls1_3" \
+            "$P_CLI debug_level=2 min_version=tls1_3 max_version=tls1_3" \
             1 \
-            -s "SSL - The requested feature is not available" \
-            -c "SSL - The requested feature is not available"
+            -s "tls13 server state: MBEDTLS_SSL_HELLO_REQUEST" \
+            -c "tls13 client state: MBEDTLS_SSL_HELLO_REQUEST"
```

Comment:
Only `MBEDTLS_SSL_HELLO_REQUEST` is available now. After client hello PR merged , other states should be check also.

Reply:
I may be wrong but I see those checks as temporary checks while we don't implement the full handshake. In that regard I think this has gone to far in the amount of code needed for that. I am referring to having a string to identify the state and all the code added to get that. A number here seems just fine for our current and temporary checks.

Reply:
ClientHello PRs have been merged , I add all state checks in `gnutls` and `openssl` test. We haven't implemented server side and I think it is too far to test other states in this test.

Also, I move `debug helpers` code to another PR, that's not part of MVP. I think it is useful and low priority.